src/main/java/samza/examples/sql/samza-sql-filter/src/main/sql/samza.sql

```diff
@@ -0,0 +1,11 @@
+-- Extract some fields from PageViewEvent and insert into a table.
```

Comment:
Could you add this example instead ? Let's avoid espresso.

-- Filter Profile change-capture stream by 'Product Manager'
--  title and project basic profile data to a kafka topic.

INSERT INTO kafka.ProductManagerProfiles
     SELECT memberId, firstName, lastName, company
     FROM kafka.ProfileChanges
     WHERE standardize(title) = 'Product Manager'

Reply:
done